MekHQ/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig: https://EditorConfig.org
+
+[*]
+charset = utf-8
+
+[*.java]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
`insert_final_newline` and `trim_trailing_whitespace` should ideally be in `[*]` rather than in `[*.java]` and `[*.xml]`.

Reply:
Good point, addressed!